src/main/java/karuna/karuna_backend/DTO/UserDTO.java

```diff
@@ -1,6 +1,10 @@
 package karuna.karuna_backend.DTO;
 
-import lombok.*;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
```

Comment:
next time to not use organize imports on all project files, just edited, changes like this should be in other pull request